engine/src/main/scala/cromwell/engine/workflow/lifecycle/execution/EngineJobExecutionActor.scala

```diff
@@ -175,11 +228,19 @@ class EngineJobExecutionActor(replyTo: ActorRef,
       stay
   }
 
-  private def recordHashError(reason: Throwable) = {
-    log.error(reason, "{}: hash error: {}. Disabling call caching for this job", tag, reason.getMessage)
+  private def disableCallCaching(reason: Throwable) = {
+    log.error(reason, "{}: Hash error, disabling call caching for this job.", jobTag)
     effectiveCallCachingMode = CallCachingOff
   }
 
+  private def disableCacheWrite(reason: Throwable) = {
+    log.error(reason, "{}: Hash error trying to retrieve or copy cache hit.", jobTag)
```

Comment:
yeah I refactored this into a function; when it was inline that wasn't so presumptuous but you're right it is now